XenAdmin/Dialogs/UpsellDialog.resx

```diff
@@ -178,7 +178,7 @@
     <value>12, 12, 12, 12</value>
   </data>
   <data name="$this.Text" xml:space="preserve">
-    <value>[XenCenter]</value>
+    <value>text</value>

```

Comment:
This seems to be a default of some sort. Will the upsell dialog show `"text"` as a label caption?